genisys/main.py

```diff
@@ -40,6 +36,9 @@ def run(subcommand, args, module):
     #nat
     nat = nt.Nat(args)
 
+    #kernelparameter
+    kernelParameter = kp.KernelParameter(args)
```

Comment:
This needs to be passed an instance of `genisys.configParser.YAMLParser`. Running it locally I get this exception thrown.
```
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/home/robert/Development/genisys/genisys/main.py", line 71, in <module>
    main()
  File "/home/robert/Development/genisys/genisys/main.py", line 68, in main
    run(args.command, args, modules)
  File "/home/robert/Development/genisys/genisys/main.py", line 31, in run
    netplan = net.Netplan(args)
              ^^^^^^^^^^^^^^^^^
  File "/home/robert/Development/genisys/genisys/modules/netplan.py", line 17, in __init__
    self.config = config.getSection("Network")
                  ^^^^^^^^^^^^^^^^^
AttributeError: 'Namespace' object has no attribute 'getSection'
```
Ditto for `nat` and `netplan` and any other modules you're making an instance of.